.ci/.e2e-tests-for-elastic-agent.yaml

```diff
@@ -1,44 +1,51 @@
 ---
+MACHINES:
+  stack:
+    image: "ami-0d90bed76900e679a"
+    instance_type: "c5.4xlarge"
+  helm:
+    image: "ami-0d90bed76900e679a"
+    instance_type: "c5.2xlarge"
+  fleet_arm64:
+    image: "ami-06dac44ad759182bd"
+    instance_type: "a1.large"
+  fleet_amd64:
+    image: "ami-0d90bed76900e679a"
+    instance_type: "c5.4xlarge"
+  fleet_elastic_pkg:
+    image: "ami-0d90bed76900e679a"
+    instance_type: "c5.4xlarge"
+  kubernetes_autodiscover:
+    image: "ami-0d90bed76900e679a"
+    instance_type: "c5.4xlarge"
+
 SUITES:
   - suite: "fleet"
-    platforms:
-      - labels: "arm64"
-      - labels: "ubuntu-18.04"
-        provider: "elastic-package"
     scenarios:
       - name: "Fleet"
-        pullRequestFilter: " && ~debian"
         tags: "fleet_mode_agent"
-        platforms:
-          - labels: "ubuntu-18.04"
-            provider: "elastic-package"
+        platforms: ["fleet_arm64", "fleet_amd64"]
```

Comment:
It's weird that we reference a MACHINES[key] with `platforms`. Maybe `machines`? or rename the top object to `PLATFORMS`?